src/plugins/imagery/Imagery_View.js

```diff
@@ -1,8 +1,8 @@
-import ImageryViewLayout from './components/ImageryViewLayout.vue';
+import ImageryView from './components/ImageryView.vue';
 
 import Vue from 'vue';
 
-export default class ImageryView {
+export default class Imagery_View {
```

Comment:
Let's just reference the ImageryView component as ImageryViewComponent in this file so we can remain consistent with our code standards.

Reply:
Done